test/test_evaluators.py

```diff
@@ -79,7 +120,7 @@ def test_wrong_one_image(self):
 
         evaluator.add_predictions(predictions, targets)
         report = evaluator.get_report()
-        self.assertEqual(report["mAP_50"], 0.75)
+        self.assertEqual(0.75, report["mAP_50"])
```

Comment:
why changing the order? the official document seems to be opposite https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual